cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -792,47 +794,47 @@ private TsvDquId buildDquId(AV_TSV_DQU tsvView, Record jrecord) {
                 jrecord.getValue(tsvView.DATE_TIME.getName(), Timestamp.class));
     }
 
-    public void create(TimeSeries input) {
+    @Override
+    public void create(TimeSeries input){
+        create(input,0, 0, 0, false, true, null,
+                true, StoreRule.REPLACE_ALL, true);
+    }
+
+    @Override
+    public void create(TimeSeries input, int utcOffsetMinutes, int intervalForward, int intervalBackward,
+                       boolean versionedFlag, boolean activeFlag, Timestamp versionDate,
+                       boolean createAsLrts, StoreRule replaceAll, boolean overrideProtection){
         connection(dsl, connection -> {
             CwmsDbTs tsDao = CwmsDbServiceLookup.buildCwmsDb(CwmsDbTs.class, connection);
 
-            int utcOffsetMinutes = 0;
-            int intervalForward = 0;
-            int intervalBackward = 0;
-            boolean versionedFlag = false;
-            boolean activeFlag = true;
             BigInteger tsCode = tsDao.createTsCodeBigInteger(connection, input.getOfficeId(),
-                    input.getName(),
-                    utcOffsetMinutes, intervalForward, intervalBackward, versionedFlag, activeFlag);
+                    input.getName(), utcOffsetMinutes, intervalForward, intervalBackward, versionedFlag, activeFlag);
```

Comment:
The utcOffsetMinutes was already set to 0 - that isn't new to these changes.   It could still be wrong but it isn't new.  These changes add those method args to the Dao interfaces and add them as optional query params to the controller methods.   

It looks like the TimeSeries object already has an interval-offset field - how about I just forward that along to the create procedure?  That way users can provide null if they want or provide a value if they want.

Reply:
I had to remove it to fix the LRL issue. you can can't a 1day regular timeseries at 6am if the interval-utc-offset is forced to 0. So not it's not new, but it was breaking operations and shouldn't be carried forward.

Reply:
I'm also not entirely sure it's necessary to create the TS first. vs just calling store the first time. Only the location already needs to exist.

Reply:
That's what I was going to say, the store_ts routine will create the ts code for you.

Reply:
That's why we ended up needing that "create_as_lrts" parameter, it couldn't distinguish between PRTS and LRTS based on the interval identifier alone.

Reply:
Ok, do we care about the intervalForward/Backward and the active flag?   I added them because it looked like create could handle it but I can drop them from the TimeSeriesController and if the user wants to tweak those they'd go through the TimeSeriesIdentifierDescriptor that I'm writing for #268

Reply:
I don't have a use-case currently for the intervalForward/Backward from the value storage. Makes sense to include them for the TimeSeriesIdentifierDescriptor.